src/css/tools.css

```diff
@@ -31,33 +31,39 @@
 
 .tool-paint-bucket .drawing-canvas-container:hover,
 .tool-colorswap .drawing-canvas-container:hover {
-  cursor: url(../img/cursors/paint-bucket.png) 12 12, pointer;
+  cursor: url(../img/cursors/paint-bucket.png) 14 14, pointer;
 }
 
 .tool-vertical-mirror-pen .drawing-canvas-container:hover {
   cursor: url(../img/cursors/vertical-mirror-pen.png) 5 15, pointer;
 }
 
-.tool-pen .drawing-canvas-container:hover,
-.tool-lighten .drawing-canvas-container:hover,
+.tool-pen .drawing-canvas-container:hover {
+  cursor: url(../img/cursors/pen.png) 1 14, pointer;
+}
+
 .tool-dithering .drawing-canvas-container:hover {
-  cursor: url(../img/cursors/pen.png) 0 15, pointer;
+  cursor: url(../img/cursors/dither.png) 1 14, pointer;
+}
+
+.tool-lighten .drawing-canvas-container:hover {
+  cursor: url(../img/cursors/lighting.png) 1 15, pointer;
```

Comment:
Could you rename lighting.png to lighten.png to be consistent with the current tool name?